portal/templates/dashboard/_macros/dashboard-facilities-table.njk

```diff
@@ -4,135 +4,171 @@
+            {{ facility.name if facility.name else 'Not entered'}}
+          </a>
+        {% else %}
+          <span data-cy="facility__name--text--{{ facility._id }}"> {{ facility.name }} </span>
+        {% endif %}
+      {% endset %}
+
+      {% if facility.hasBeenIssued %}
+        {% set stage = 'Issued' %}
+      {% else %}
+        {% set stage = 'Unissued' %}
+      {% endif %}
+
+      {# {% if facility.currency.id and facility.value %}
+        {% set currency = facility.currency.id %}
+        {% set value = currency + ' ' + facility.value %}
+      {% endif %} #}
+      {% set currencyAndValue = '-' %}
+
+      {% if facility.currency.id and facility.value %}
+        {% set formattedValue = facility.value | formatAsCurrency %}
+        {% set currencyAndValue = facility.currency.id + ' ' + formattedValue %}
+      {% endif %}
+
+      {% set facilityRow = [
+        {
+          text: facility.exporter,
+          attributes: { "data-cy": "facility__exporter--" + facility._id  },
+          classes: "govuk-body-s"
+        },
+        {
+          html: linkHtml,
+          attributes: { "data-cy": "facility__bankId--" + facility._id  },
+          classes: "govuk-body-s"
+        },
+        {
+          text: facility.ukefFacilityId | dashIfEmpty,
+          attributes: { "data-cy": "facility__ukefId--" + facility._id },
+          classes: "govuk-body-s"
+        },
+        {
+          text: facility.type | dashIfEmpty,
+          attributes: { "data-cy": "facility__type--" + facility._id },
+          classes: "govuk-body-s"
+        },
+        {
+          text: facility.submissionType | dashIfEmpty,
+          attributes: { "data-cy": "facility__noticeType--" + facility._id },
+          classes: "govuk-body-s"
+        },
+        {
+          text: currencyAndValue,
+          attributes: { "data-cy": "facility__value--" + facility._id },
+          classes: "govuk-body-s"
+        },
+        {
+          text: stage  | dashIfEmpty,
+          attributes: { "data-cy": "facility__bankStage--" + facility._id },
+          classes: "govuk-body-s"
+        },
+        {
+          text: facility.submittedAsIssuedDate | localiseTimestamp("D MMM YYYY", userTimezone) | dashIfEmpty,
+          attributes: { "data-cy": "facility__issuedDate--" + facility._id },
+          classes: "govuk-body-s"
         }
-      },
-      {
-        text: "Issued date",
-        attributes: {
-          "data-cy": "facility__date--header"
+      ] %}
+
+      {% set rows = (rows.push(facilityRow), rows) %}
+    {% endfor %}
+
+    {% set exporterHTML %}
+      {% set buttonValue = 'ascending' %}
+
+      {% if activeSortByOrder === 'ascending' %}
+        {% set buttonValue = 'descending' %}
+      {% endif %}
+
+      <button
+        type="submit"
+        name="sortBy"
+        value="{{ buttonValue }}"
+        data-cy="facility__exporter--header-button">
+        Exporter
```

Comment:
Is `Exporter` available in `{{ buttonValue }}` variable?

Reply:
no, it is only what kind of sort such as ascending or descending